src/contacts/contacts.ts

```diff
@@ -36,7 +36,7 @@ export class Contacts {
     const data = await this.resend.post<CreateContactResponseSuccess>(
       `/audiences/${payload.audienceId}/contacts`,
       {
-        ...payload,
+        unsubscribed: payload.unsubscribed,
```

Comment:
this one needs `email` as well
```suggestion
        unsubscribed: payload.unsubscribed,
        email: payload.email,
```